regulations/tasks.py

```diff
@@ -19,11 +19,18 @@
 
 logger = get_task_logger(__name__)
 
+# The following limit is specified by the regulations.gov API
+# It is not available as a queryable endpoint
+MAX_ATTACHMENT_COUNT = 10
 
-@shared_task
-def submit_comment(body):
+
+@shared_task(bind=True)
+def submit_comment(self, body):
     comment = build_comment(body)
     files = extract_files(body)
+    if len(files) > MAX_ATTACHMENT_COUNT:
```

Comment:
Can we check the attachment count in `regulations/views/comment:submit_comment` instead? Then we can send back a json error with a 422 status. Then we'd never even queue the submit task with too many attachments.